src/backend/base/langflow/base/data/utils.py

```diff
@@ -143,7 +143,7 @@ def parse_text_file_to_data(file_path: str, silent_errors: bool) -> Data | None:
         elif file_path.endswith((".yaml", ".yml")):
             text = yaml.safe_load(text)
         elif file_path.endswith(".xml"):
-            xml_element = ET.fromstring(text)
+            xml_element = ET.fromstring(text)  # noqa: S314
```

Comment:
Is this data trusted ? [Rule S314](https://docs.astral.sh/ruff/rules/suspicious-xml-element-tree-usage/) suggests to use the safer lib defusedxml.

Reply:
The data comes from the user, which means if they load untrusted data, it will damage their system. If defusedxml works just as well, then I say we use it instead.

Reply:
added defusedxml.